### PR TITLE
upgraded and consolidated js datatables(and extensions) packages

### PR DESCRIPTION
**Pull request by bradymiller**
TODO:
Test the Zend patient validator.
Look through the bower packages to ensure nothing needs to be remove (.gitignore stuff etc.)

### CODE REVIEW

File: interface/main/finder/dynamic_finder.php [hunk 1]
```diff
@@ -43,16 +43,12 @@
     <title><?php echo xlt("Patient Finder"); ?></title>
 <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
 
-<style type="text/css">
-@import "../../../library/js/datatables/media/css/demo_page.css";
-@import "../../../library/js/datatables/media/css/demo_table.css";
-.mytopdiv { float: left; margin-right: 1em; }
```
Comment (bradymiller) [inline interface/main/finder/dynamic_finder.php]: may need to add the above mytopdiv style line back

Comment (bradymiller) [inline interface/main/finder/dynamic_finder.php]: didn't do anything on testing, so left it out

**Comment by bradymiller** [comment]
Testing well, so will bring this in. Will also bring into rel-500, since this is a bug fix for the Zend patient validator module.

**Comment by sunsetsystems** [comment]
Cool.  I'm a bit surprised this works since Datatables 1.10 changed the whole API.

**Comment by bradymiller** [comment]
The only hang up was actually the newer jquery verion which no longer supported the .live function.

**Pull request merged**